src/cli/hpr_cli_config.erl

```diff
@@ -105,15 +105,16 @@ config_oui_list(["config", "oui", OUIString], [], []) ->
         NetID = hpr_route:net_id(Route),
         Server = hpr_route:server(Route),
 
-        DevAddrRanges = lists:map(FormatDevAddr, hpr_route:devaddr_ranges(Route)),
+        DevAddrRanges = lists:map(
+            FormatDevAddr, hpr_route_ets:devaddr_ranges_for_route(hpr_route:id(Route))
+        ),
         DevAddrInfo = [DevAddrHeader | DevAddrRanges],
 
-        EUIs = lists:map(FormatEUI, hpr_route:euis(Route)),
+        EUIs = lists:map(FormatEUI, hpr_route_ets:eui_pairs_for_route(hpr_route:id(Route))),
```

Comment:
If this is going to be useful, we might to put this behind a flag. 
This will easily be a few thousand for router operator Routes.